### PR TITLE
add fluent-bit

### PR DESCRIPTION
**Pull request by cw-ozaki**
I made a helm chart of [Fluent Bit](https://fluentbit.io/).
There is a [stable/fluent-bit](https://github.com/helm/charts/tree/master/stable/fluent-bit), but it is difficult to support BigQuery, CloudWatch and S3, so I decided to create a new one.

#### Checklist

- [x] Chart Version bumped

### CODE REVIEW

File: fluent-bit/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
```
Comment (cw-sakamoto) [inline fluent-bit/Chart.yaml]: https://helm.sh/docs/topics/charts/
>The apiVersion field should be v2 for Helm charts that require at least Helm 3

If `apiVersion` is v2,  dose makefile work ? 
https://github.com/chatwork/charts/pull/146/files#diff-8787ed5594febb4855b1641633af1e8cR6

File: fluent-bit/templates/daemonset.yaml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+              mountPath: /secure/{{ $name }}
+              subPath: {{ $name }}
+            {{- end }}
+      volumes:
+        - name: varlog
+          hostPath:
+            path: /var/log
+        - name: varlibdockercontainers
+          hostPath:
+            path: /var/lib/docker/containers
```
Comment (cw-sakamoto) [inline fluent-bit/templates/daemonset.yaml]: I think it's better to deal with psp.
https://kubernetes.io/docs/concepts/policy/pod-security-policy/
https://github.com/helm/charts/blob/master/stable/fluent-bit/templates/psp.yaml
https://github.com/helm/charts/blob/master/stable/fluent-bit/templates/cluster-role.yaml#L18-L27

File: fluent-bit/templates/serviceaccount.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{{- if .Values.serviceAccount.create }}
```
Comment (cw-sakamoto) [inline fluent-bit/templates/serviceaccount.yaml]: This way of writing makes it possible to create only serviceaccount, but what kind of use case ?
FLY: https://github.com/helm/charts/pull/21204/files#diff-a196c8e95b4eb0a3e01b1b782632008bR1

File: fluent-bit/Chart.yaml (hunk 1 above)
Comment (cw-ozaki) [reply fluent-bit/Chart.yaml]: Looks like CI is through😁
https://circleci.com/gh/chatwork/charts/1150?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: fluent-bit/templates/serviceaccount.yaml (hunk 3 above)
Comment (cw-ozaki) [reply fluent-bit/templates/serviceaccount.yaml]: @cw-sakamoto 
I don't have a use case.
However, in the latest version, it looks like this when created with `helm create`, so we haven't changed it for clarity.

File: fluent-bit/templates/daemonset.yaml (hunk 2 above)
Comment (cw-ozaki) [reply fluent-bit/templates/daemonset.yaml]: @cw-sakamoto 
https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/policy/restricted-psp.yaml
I created the PSP by referring to the link. (https://github.com/chatwork/charts/pull/146/commits/93efee6bfd4ddc6d5255d68de6ec3ac251e09ada)
I only gave permissions around the volume based on what was restricted.

**Pull request merged**